### ISSUE TITLE
Add sync LRO sample

### ISSUE THREAD

**Issue by tg-msft** [Azure/azure-sdk-for-net#14757]
Since we don't have a sync overload of `WaitForCompletionAsync`, it would be great to add a sample to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/LongRunningOperations.md that explains the best practices and trade-offs for waiting on completion of an LRO in a sync method.

**Comment by JimSuplizio** [comment]
Hi @tg-msft, we deeply appreciate your input into this project. Regrettably, this issue has remained inactive for over 2 years, leading us to the decision to close it. We've implemented this policy to maintain the relevance of our issue queue and facilitate easier navigation for new contributors. If you still believe this topic requires attention, please feel free to create a new issue, referencing this one. Thank you for your understanding and ongoing support.
